curr_scraper/settings.py

```diff
@@ -24,18 +26,19 @@
 # See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-(mrb(mbnraywr(=4im+ux7%0v+0ia36)0$mk5ox&_(yex5_f!t'
+SECRET_KEY = os.environ.get('SECRET_KEY')
+
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 ALLOWED_HOSTS = []
 
 
-
 # Application definition
 
 INSTALLED_APPS = [
+    'api_caller.apps.ApiCallerConfig',
```

Comment:
Please change the INSTALLED_APPS structure to what the cookiecutter project uses: DJANGO_APPS, THIRD_PARTY_APPS, LOCAL_APPS, and INSTALLED_APPS.

Here's an example from their source files: https://github.com/cookiecutter/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/config/settings/base.py